universe/worlds/api_fly_with_me.go

```diff
@@ -0,0 +1,187 @@
+package worlds
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+
+	"github.com/momentum-xyz/posbus-protocol/posbus"
+	"github.com/momentum-xyz/ubercontroller/universe/common/api"
+	"github.com/momentum-xyz/ubercontroller/universe/common/api/dto"
+)
+
+// @Summary Starts a fly with me session for a certain world
+// @Schemes
+// @Description Initiates a forced fly with me session for all users in a given world
+// @Tags worlds
+// @Accept json
+// @Produce json
+// @Param world_id path string true "World ID"
+// @Success 200 {object} nil
+// @Failure 500 {object} api.HTTPError
+// @Failure 400 {object} api.HTTPError
+// @Failure 404 {object} api.HTTPError
+// @Router /api/v4/worlds/{world_id}/fly-with-me/start [post]
+func (w *Worlds) apiWorldsFlyWithMeStart(c *gin.Context) {
+	worldID, err := uuid.Parse(c.Param("worldID"))
+	if err != nil {
+		err := errors.WithMessage(err, "Worlds: apiWorldsFlyWithMeStart: failed to parse world id")
+		api.AbortRequest(c, http.StatusBadRequest, "invalid_world_id", err, w.log)
+		return
+	}
+
+	world, ok := w.GetWorld(worldID)
+	if !ok {
+		err := errors.Errorf("Worlds: apiWorldsFlyWithMeStart: world not found: %s", worldID)
+		api.AbortRequest(c, http.StatusNotFound, "world_not_found", err, w.log)
+		return
+	}
+
+	token, err := api.GetTokenFromContext(c)
+	if err != nil {
+		err = errors.WithMessage(err, "Worlds: apiWorldsFlyWithMeStart: failed to get token from context")
+		api.AbortRequest(c, http.StatusInternalServerError, "failed_to_get_token", err, w.log)
+		return
+	}
+
+	userID, err := api.GetUserIDFromToken(token)
+	if err != nil {
+		err = errors.WithMessage(err, "Worlds: apiWorldsFlyWithMeStart: failed to get user id from token")
+		api.AbortRequest(c, http.StatusInternalServerError, "failed_to_get_user_id", err, w.log)
+		return
+	}
+
+	user, ok := world.GetUser(userID, true)
+	if !ok {
+		err := errors.Errorf("Worlds: apiWorldsFlyWithMeStart: user not present in world: %s", worldID)
+		api.AbortRequest(c, http.StatusNotFound, "user_not_found", err, w.log)
+		return
+	}
+
+	userProfile, err := w.db.UsersGetUserProfileByUserID(c, user.GetID())
+	if err != nil {
+		err = errors.WithMessage(err, "Worlds: apiWorldsFlyWithMeStart: failed to get user profile by user id")
+		api.AbortRequest(c, http.StatusNotFound, "profile_not_found", err, w.log)
+		return
+	}
+
+	var userName = ""
```

Comment:
cosmetic change: from what I have read and done as a practice I use `var` when I just init a variable and `:=` when assigning to it. Since here you initialize to the zero value of the type  `var userName string` can be most clear , but `userName := ""` is also fine I guess. @cnaize can give a final word on how we format these things and if it should really matter at all and do such changes.